pipeline/bicleaner/packs.py

```diff
@@ -1,18 +1,22 @@
 import requests
 
-bicleaner = 'https://github.com/bitextor/bicleaner-data/releases/latest/download'
-bicleaner_ai = 'https://github.com/bitextor/bicleaner-ai-data/releases/latest/download'
+bicleaner = "https://github.com/bitextor/bicleaner-data/releases/latest/download"
```

Comment:
black is a very opinionated formatter (see https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html for details on the style choices).

Reply:
Part of the workflow for using opinionated formatters is that they auto-format things for you so that you don't have to worry about consistency at the code review time. Ideally they are integrated directly into your IDE. You can also add them as pre-commit hooks although personally I don't like this approach for when I'm doing rebasing and things.

Reply:
ok, they explain it here https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings mostly with unification and similarity to C... I'm gonna miss single quotes, I love them so much :)